src/pudl/glue/ferc1_eia.py

```diff
@@ -289,13 +289,10 @@ def plants_ferc1_raw(**transformed_plant_tables):
         )
         return all_plants
 
-    tfr_mapping = {
-        table_name: GenericPlantFerc1TableTransformer for table_name in plant_tables
-    }
     transform_assets = [
         ferc1_transform_asset_factory(
             table_name,
-            tfr_mapping,
+            GenericPlantFerc1TableTransformer,
```

Comment:
this changed bc moved the dict of tbl_name (key): transformer (value) out of `create_ferc1_transform_assets` so now `ferc1_transform_asset_factory` takes the specific transformer instead of the full dict. i think this is a nice simplification overall!